### DISCUSSION TITLE
Supporting only using 2 modifier keys (alt and shift are modifiers) and no other keys

### DISCUSSION THREAD

**Discussion by refaelsh** [qtile/qtile#3706] [Ideas] [answered]
I want to be able to switch keyboard layout with Alt+Shift, as far as I understood from here: https://github.com/qtile/qtile/issues/1372#issuecomment-508663271, it is not currently possible to use 2 modifier keys (alt and shift are modifiers) and no other keys.

**Comment by elParaguayo** [comment, marked as answer]
Do something like:
``` python
Key(["mod1"], "Shift_L", lazy.spawn("cmd to change keyboard"))
```
`mod1` is left Alt for me.

**Reply by refaelsh** [reply]
Thank you. That works.
